web/netlify.toml

```diff
@@ -37,5 +37,11 @@ to = "/index.html"
 status = 200
 
 
+[[redirects]]
+from = "/handbook"
+to = "docs.dandiarchive.org"
+status = 200
```

Comment:
i don't know what status means over here, but should be a `308` i think for permanent redirect which will tell search engines and bots to forget the source if they do so.

Reply:
Thanks.  I have updated.

Reply:
We may want `301`, since it has better compliance from clients and its limitation (that it converts other methods into `GET` upon redirect) is fine for us, since all requests should be `GET` anyway.